Modularize and clarify LB architecture & considerations (#2940)
theforeman/foreman-documentation: guides/common/modules/con_additional-maintenance-required-for-load-balancing.adoc

[id="additional-maintenance-required-for-load-balancing_{context}"]
= Additional maintenance required for load balancing

Configuring {SmartProxies} to use a load balancer results in a more complex environment and requires additional maintenance.

The following additional steps are required for load balancing:

> * You must ensure that all {SmartProxies} have the same content views and synchronize all {SmartProxies} to the same content view versions.

Reviewer: This is unclear to me; esp. the second half of the sentence.

Author: Do I read this correctly that line 8 is simply not valid anymore in the latest version and I can remove it? @jeremylenz

Reviewer 2: The opposite; line 8 became valid recently.

Author: Strange, it looks like that requirement has been documented [at least since 2020](https://github.com/theforeman/foreman-documentation/commit/c84f2a8afcb). Ok, I'll leave this conversation open to give @sjha4 and/or @maximiliankolb space to review and comment if needed.

Reviewer 2: Ah, I see.. It could also be interpreted to mean "If you publish a new content view version to a synced lifecycle environment, you must then re-sync all of your smart proxies so that they all continue to have the same content."

Reviewer 3: I read that to understand that all smart proxies behind LB should have the same content so balancing can work correctly..

Reviewer: ```suggestion
* You must ensure that all {SmartProxies} have the same content.
If you publish a content view version on {Project}, synchronize it to all {SmartProxyServers}.
```

Do you sync content to Smart Proxy Servers via Infrastructure > Smart Proxies > my Smart Proxy Server > overview tab > synchronize? Or is this done automatically?

* You must upgrade each {SmartProxy} in sequence.

> * You must back up each {SmartProxy} that you configure regularly.

Reviewer: what does "regularly" mean? If I do not use LB, can I skip backups?

Maybe something like "backup all SP behind LB even if you think their identical" in different words?

Author: This looks like content that has been here for years so based on the comment above, let's try removing it.
```suggestion
```
